Logout, age and passwords match

Pokerek opened the pull request:
Change match validation for passwords
Add min age validation to 18
Add logout action to profile
Change matches list to table
Add age to profile view

Pokerek committed:
Logout, age and passwords match
Change match validation for passwords
Add min age validation to 18
Add logout action to profile
Change matches list to table
Add age to profile view

graddaniel reviewed (changes requested):
Sprawdzenie haseł można zrobić prościej

Pokerek committed:
Change register action logic
Add Action result class
Move localStorage action with token to service
Change maxWidth in profile table
